src/components/backend-ai-login.ts

```diff
@@ -467,13 +467,16 @@ export default class BackendAILogin extends BackendAIPage {
   }
 
   private _getConfigValueByExists(parentsKey, valueObj: ConfigValueObject) {
+    const uncommentedValue =
+      typeof valueObj.value === 'string' && valueObj.value.includes('#')
+        ? valueObj.value.split('#')[0].trim()
+        : '';
```

Comment:
This is not a good idea because the value can have a #. 
```
some="This#Value#has#"
```

Reply:
If the current TOML parser does not support inline comments, please search for an alternative one that supports inline comments.